python/GafferUI/PathListingWidget.py

```diff
@@ -1028,6 +1081,108 @@ def __sectionResized( self, index, oldWidth, newWidth ) :
 		column = self.__getColumn( index )
 		self.__columnWidthAdjustments[column] = newWidth - self.__idealColumnWidths[column]
 
+		# When any column is user resized, automatically adjust the last column's width
+		# in order to make use of any empty space.
+		if index == self.header().count() - 1 :
+			# We prevent the last column from being user resized narrower than the
+			# available empty space in the header, but still allow it to be enlarged
+			# by providing the user specified column width as the minimum.
+			self.__resizeLastColumnToAvailableWidth( newWidth )
+		else :
+			self.__resizeLastColumnToAvailableWidth()
+
+	def __resizeLastColumnToAvailableWidth( self, minimumSuggestedWidth = 0 ) :
+
+		header = self.header()
+		if header.stretchLastSection() :
+			# Qt's behaviour will override and extend the last section for us automatically
+			return
```

Comment:
Is this redundant, since we call `setStretchLastSection()` in `_TreeView.__init__`?

Reply:
Indeed, redundant now we're forcing that off in `__init__`. Removed in 63a7d55.